docs/cloud/cloud-tables/cloud-table-upload-data.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Upload data
+layout: default
+parent: Manage tables
+grand_parent: Cloud
+nav_order: 2
+---
+
+# How do I upload data to FeatureBase Cloud?
+{: .no_toc }
+
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Create a database](/docs/cloud/cloud-databases/cloud-db-create)
+
+## Naming standards
+
+{% include /concepts/standard-naming-obj.md%}
+{% include /cloud-table/cloud-standard-naming-table.md %}
+
+## Step 1: Navigate to Upload Data
+
+| Click **Databases** > database name > **Upload Data**. |
+
+## Step 2: Select a file to upload
+
+1. Click **Upload a file**.
+2. Select a file from the file browser.
+3. Click **Done**.
+
+{: .note }
```

Comment:
This needs to be stated before the instructions between Naming Standards and Step 1

The user needs to know this information BEFORE they try uploading anything.

```
## Upload file

Your CSV data file needs the following structure:
* Header row with first item the unique identifier for each row
* comma-separated values for each column
```